src/client/components/icons/ArrowBack.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { IconRoot, IconRootProps } from './IconRoot'
+
```

Comment:
I was thinking about that as well but kind of liked the explicitness of having to for different directions (and actually it's not exactly the same - the `Arrow` we have are thicker than the one in the Icon Library in Figma. So that should be replaced). 

But then, having a direction prop also opens up for other directions, instead of adding a new icon every time we want an arrow in another direction. So maybe you're right. I'll look into it.

Reply:
👍 I hear you, sometimes it's more work for a small benefit. Either way works. 🙌

Reply:
Ah, interesting 👀

Reply:
Will merge this now though and look at this for another PR :)